packages/dd-trace/src/constants.js

```diff
@@ -28,6 +28,7 @@ module.exports = {
   CLIENT_PORT_KEY: 'network.destination.port',
   PEER_SERVICE_KEY: 'peer.service',
   PEER_SERVICE_SOURCE_KEY: '_dd.peer.service.source',
+  PEER_SERVICE_REMAP_KEY: '_dd.peer.service.remapped_from',
```

Comment:
Is this a standard across tracers? `_dd.peer.service.orig` might be more succinct ;)

Reply:
Yes, it's standard across tracers.